### PR TITLE
Lagom 1.5 RC2 continued

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Comment by johanandren** [comment]
Yeah, I'm messing around with travis and jdk 11 in another pr, so I'll get that done so that this can be validated before we merge it.

File: lagom-persistence-couchbase/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/couchbase/PersistentEntityRefSpec.scala [hunk 1]
```diff
@@ -87,6 +82,7 @@ class PersistentEntityRefSpec
     override def materializer: Materializer = ActorMaterializer()(system)
     override def serviceLocator: ServiceLocator = NoServiceLocator
     override def jsonSerializerRegistry: JsonSerializerRegistry = TestEntitySerializerRegistry
+    override def coordinatedShutdown: CoordinatedShutdown = CoordinatedShutdown(system)
```
Comment (TimMoore) [inline lagom-persistence-couchbase/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/couchbase/PersistentEntityRefSpec.scala]: The build is failing with:

```
[error] /home/travis/build/akka/akka-persistence-couchbase/lagom-persistence-couchbase/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/couchbase/PersistentEntityRefSpec.scala:90:18: method coordinatedShutdown overrides nothing
[error]     override def coordinatedShutdown: CoordinatedShutdown = CoordinatedShutdown(system)
[error]                  ^
[error] one error found
[info] Done compiling.
[error] (lagom-persistence-couchbase-scaladsl / Test / compileIncremental) Compilation failed
```

File: project/Dependencies.scala [hunk 2]
```diff
@@ -11,7 +11,7 @@ object Dependencies {
 
   val AkkaVersion = "2.5.21"
   val AlpakkaCouchbaseVersion = "1.0-M3"
-  val LagomVersion = "1.5.0-M2"
+  val LagomVersion = "1.5.0-RC1"
```
Comment (TimMoore) [inline project/Dependencies.scala]: Shouldn't it be RC2?

File: lagom-persistence-couchbase/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/couchbase/PersistentEntityRefSpec.scala (hunk 1 above)
Comment (TimMoore) [reply lagom-persistence-couchbase/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/couchbase/PersistentEntityRefSpec.scala]: I think I get it now… this maybe is needed for RC2 but not RC1

File: project/Dependencies.scala (hunk 2 above)
Comment (johanandren) [reply project/Dependencies.scala]: Meh, it should indeed, rebase miss.

**Pull request merged**